hwdb.d/60-keyboard.hwdb

```diff
@@ -916,7 +916,7 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP:pnHPEliteDragonfly13.5inchG3NotebookPC:pvr*
 
 # Huawei WMI hotkeys driver
 evdev:name:Huawei WMI hotkeys:dmi:bvn*:bvr*:bd*:svnHUAWEI:*
- KEYBOARD_KEY_287=f20                                   # Microphone mute button, should be micmute
+ KEYBOARD_KEY_287=micmute                               # Microphone mute button
```

Comment:
This mapping has been the default mapping of the kernel driver since the driver was introduced, so this entire section can simply be dropped now.